lib/src/rules/avoid_redundant_argument_values.dart

```diff
@@ -8,11 +8,11 @@ import 'package:analyzer/dart/element/element.dart';
 
 import '../analyzer.dart';
 
-const _desc = r'Avoid redundant parameter values.';
+const _desc = r'Avoid redundant argument values.';
 
-const _details = r'''Avoid redundant parameter values.
+const _details = r'''Avoid redundant argument values.
 
-**DON'T** declare parameters with values that match their defaults.
+**DON'T** declare argument with values that match their defaults.
```

Comment:
nit: Perhaps "their defaults" --> "the defaults for the corresponding parameter"?

Reply:
👍